packages/core/utils/lib/sanitize/sanitizers.js

```diff
@@ -20,17 +20,20 @@ const {
   removeMorphToRelations,
 } = require('./visitors');
 
-const sanitizePasswords = curry((schema, entity) => {
+const sanitizePasswords = (schema) => async (entity) => {
   return traverseEntity(removePassword, { schema }, entity);
-});
-
-const sanitizePrivates = curry((schema, entity) => {
-  return traverseEntity(removePrivate, { schema }, entity);
-});
+};
 
-const defaultSanitizeOutput = curry((schema, entity) => {
-  return pipeAsync(sanitizePrivates(schema), sanitizePasswords(schema))(entity);
-});
+const defaultSanitizeOutput = async (schema, entity) => {
+  return traverseEntity(
+    (...args) => {
+      removePassword(...args);
+      removePrivate(...args);
```

Comment:
probably not a problem, but in another PR we mentioned that in some cases if the first fucntion removes the key, the second could throw an error if it is expecting it.

Reply:
Yeah we would need a bigger refactor to support it correctly but the most effective perf gains without refactoring is to groupe the operation in one visitor to avoid running the traversal multiple times :) 

Here I know we can't have a conflict but that will be something to be carefull about until we refactor the code more